sdk/servicebus/service-bus/test/receiveAndDeleteMode.spec.ts

```diff
@@ -344,7 +344,7 @@ describe("Unsupported features in ReceiveAndDelete mode", function(): void {
   }
 
   const testError = (err: Error, operation: DispositionType): void => {
-    expect(err.message, "ErrorMessage is different than expected").includes(
+    expect(err.message.toLowerCase(), "ErrorMessage is different than expected").includes(
```

Comment:
.toLowercase is to hadle the casing for dead-letter.
![image](https://user-images.githubusercontent.com/10452642/76580461-47715f00-648d-11ea-928c-add8bb3ab95e.png)